tests/test_bukuDb.py

```diff
@@ -23,6 +23,10 @@
 from buku import BukuDb, parse_tags, prompt
 
 
+def _add_rec(db, *args, **kw):
+    """Use THIS instead of db.add_rec() UNLESS you want to wait for unnecessary network requests."""
```

Comment:
Using a custom wrapper is harder to miss when looking through tests or adding a new one. So, using the method directly in tests should only be done when either a) the network query is part of the test, or b) the method _is_ what's being tested (in which case explicit use is preferable).